code/modules/projectiles/guns/magic/wand.dm

```diff
@@ -60,7 +73,7 @@
 	add_attack_logs(user, user, "zapped [user.p_themselves()] with a [src]", ATKLOG_ALL)
 
 /////////////////////////////////////
-//WAND OF DEATH
+// MARK: WAND OF DEATH
 /////////////////////////////////////
```

Comment:
```suggestion
// WAND OF DEATH

```
Mark for all of these is a bit excessive, especially since you included a contents comment at the top of the file. Single-line comments here will suffice.